# describes a kinematic chain
Header header
string name
string object1
string object2
Segment[] segments

